cmd/cli/kubectl-kyverno/apply/apply_command.go

```diff
@@ -112,7 +112,7 @@ func Command() *cobra.Command {
 	var cmd *cobra.Command
 	var resourcePaths []string
 	var cluster, policyReport, stdin, registryAccess bool
-	var mutateLogPath, variablesString, valuesFile, namespace, userInfoPath string
+	var mutateLogPath, variablesString, valuesFile, namespace, userInfoPath, kubecfgpath, context string
```

Comment:
Please use camel case in kubecfgpath to make it more readable.

Reply:
Done. i thought it is better to use just kubeconfig instead of kubecfgpath.